src/TeamService/Provider.php

```diff
@@ -35,14 +35,16 @@ protected function getTokenUrl()
      */
     protected function getUserByToken($token)
     {
-        $response = $this->getHttpClient()->get(
-            'https://'.$this->getAccount().'.vssps.visualstudio.com/_apis/profile/profiles/me?api-version=5.0-preview.3',
-            [
-                RequestOptions::HEADERS => [
-                    'Authorization' => 'Bearer '.$token,
-                ],
-            ]
-        );
+        $uri = 'https://'.$this->getAccount().'.vssps.visualstudio.com/_apis/profile/profiles/me';
+
+        $response = $this->getHttpClient()->get($uri, [
+            RequestOptions::HEADERS => [
+                'Authorization' => 'Bearer '.$token,
+            ],
+            RequestOptions::QUERY => [
+                'api-version' => '5.0-preview.3',
```

Comment:
Do we need to consider any differences in URL encoding here?

Reply:
No